sources/packages/backend/apps/api/src/route-controllers/application/application.aest.controller.ts

```diff
@@ -68,15 +68,36 @@ export class ApplicationAESTController extends BaseController {
    * @returns Application details
    */
   @Get(":applicationId")
-  @ApiNotFoundResponse({ description: "Application not found." })
+  @ApiNotFoundResponse({
```

Comment:
In this controller, there is one more API that needs to be adapted for assessments. 

![image](https://github.com/user-attachments/assets/bbd8763f-0e97-4133-b800-ae79024d481d)